FrontEnd/src/components/ProfileDetail/BannerImage.module.css

```diff
@@ -0,0 +1,15 @@
+.banner-image__block {
+    width: 1512px;
```

Comment:
Does it work correctly for smaller and bigger screens?

Reply:
Horizontal scroll appears when screen is smaller. In case of bigger screen there is a white space in the right of whole page.